src/index.ts

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env node
 
-import net = require('node:net')
-import express = require('express')
-import open = require('open')
-import fs = require('node:fs')
-import fsp = require('node:fs/promises')
-const { resolve, join } = require('node:path')
-import yargs = require('yargs/yargs')
+import { SocketAddress, createConnection } from'node:net'
```

Comment:
Keep code style consistent.
```suggestion
import { SocketAddress, createConnection } from 'node:net'
```